app/src/main/res/drawable/tile_square_tv.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="rectangle">
+            <size
+                android:width="384dp"
+                android:height="384dp" />
```

Comment:
Looking at `tile_letters.xml` we currently use a size of 256x256 for square and don't prefix it with _square.

Reply:
:+1: fixed